CMakeLists.txt

```diff
@@ -16,19 +16,31 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(LXQTBT_MINIMUM_VERSION "0.6.0")
 set(QT_MINIMUM_VERSION "5.7.1")
 set(QTXDG_MINIMUM_VERSION "3.3.0")
-set(FMQT_SO_VERSION "6")
-
-add_definitions(
-    -DFMQT_SO_VERSION=\"${FMQT_SO_VERSION}\"
-)
 
 find_package(Qt5DBus ${QT_MINIMUM_VERSION} REQUIRED)
 find_package(Qt5LinguistTools ${QT_MINIMUM_VERSION} REQUIRED)
 find_package(Qt5Widgets ${QT_MINIMUM_VERSION} REQUIRED)
 find_package(Qt5XdgIconLoader ${QTXDG_MINIMUM_VERSION} REQUIRED)
 find_package(lxqt-build-tools ${LXQTBT_MINIMUM_VERSION} REQUIRED)
+find_package(fm-qt)
 find_package(dbusmenu-qt5 REQUIRED)
 
+if (fm-qt_FOUND)
+    get_target_property(LIB_FM_QT_CONFIGURATIONS fm-qt IMPORTED_CONFIGURATIONS)
```

Comment:
Please excuse my stupidity - but how can fm-qt  be found without configuration?

Reply:
You mean the conditions? I think they cover all possible scenarios.

Reply:
@agaida Didn't understood your question.

Reply:
He might mean, "What if lxqt-qtplugin is compiled without libfm-qt?" It'll be distro maintainers' responsibility if they make that mistake, I guess.

Reply:
Then it will fallback to the `libfm-qt.so` as before.